hack/app_sre_pr_check.sh

```diff
@@ -16,6 +19,12 @@ do
         echo "ERROR: Rule missing 'namespace' in file '$F'"
         MISSING_NS="true"
     fi
+
+    JSON_RULESFILE="$(cat "$F" | python -c 'import json, sys, yaml ; y=yaml.safe_load(sys.stdin.read()) ; print(json.dumps(y))' | jq -r '.spec')"
+    if ! echo "$JSON_RULESFILE" | $CONTAINER_ENGINE run -i --rm --entrypoint promtool $PROMTOOL_IMAGE check rules --lint="none"; then
```

Comment:
Is the `--lint=none` intentional?

Reply:
Yes, that's intended right now. `promtool check rules` does two things:
- verify correctness of rule files and making sure they can be parsed
- linting the rules files, making sure that they follow good practices.

The linting currently only consists of a single rule that checks for duplicate *rule* names. Duplicate rule names are, however, supported, contrary to duplicate *group* names, an error which is caught by the promtool verifying the basic correctness of the rules files.